data/hai/hai missions.txt

```diff
@@ -1500,7 +1135,66 @@ mission "Pirate Troubles [3]"
 			label tribute
 			`	"Excellent!" the pirate says, which is met with cheers from the rest of the pirates in the room. "But make it quick. Wouldn't want to keep us waiting." The blast doors reopen, and you return to your ship to leave immediately.`
 				flee
-	
+			
+			label unfettered
+			choice
+				`	"I'm <first> <last>, here to negotiate on behalf of the Unfettered Hai."`
+					goto offer
+				`	"I'm <first> <last>, here to negotiate on behalf of the Hai."`
+					goto negotiate
+				`	"I'm <first> <last>, here to take this gang down."`
+			`	The voice laughs. "Well too bad, because you're right in the middle of a hornet's nest. Fire, boys!"`
+				goto die
+			label offer
+			`	The voice laughs. "What? There are "Unfettered" squirrels too now? Fancy! I wonder what they want. Boys!"`
+			`	The floodlights turn off, revealing a large room with what must be hundreds of pirates with guns of various sizes trained on your ship. One wrong move and you're toast. "Please, Captain. Step out of your ship."`
+			choice
+				`	(Comply with the orders.)`
+					goto obey
+				`	(Attempt to escape.)`
+			`	You activate your ship's engines to escape. The floodlights suddenly reactivate.`
+				goto die
+
+			label obey
+			`	You step out of your ship, and a tall man with a scar across his cheek steps forward from the crowd of pirates, presumably the leader of the gang.`
+			choice
+				`	"The Unfettered Hai ask that you continue attacking the Hai systems."`
+					goto continue
+				`	"You must be Scar."`
+			`	"No. Scar is long dead," the leader says.`
+			choice
+				`	"Oh, so does that make you Scab?"`
+				`	"The Unfettered Hai ask that you continue attacking the Hai systems."`
+					goto continue
+			`	The leader grimaces. "I'll cut your tongue out if you don't watch your mouth."`
+			`	You take a step back. "The Unfettered Hai ask that you continue attacking the Hai systems," you say, attempting not to get shanked.`
+			label continue
+			`	The leader seems surprised. "Well that's what was planned anyway, so what are you here for then?"`
+			choice
+				`	"I am here to propose a deal, the Unfettered want your attacks to increase and are wondering how they could help."`
+					goto deal
+				`	"No particular reason; just enjoying the view."`
+			`	The leader laughs and turns to the crowd, "He's a funny one, boys!" They all laugh in response.`
+			label deal
+			`	"Listen, here's what I want: bring us Hai tech. In exchange we will continue doing what we do, which is annoying the hell out of squirrels!"`
+			choice
+				`	"That seems like a good deal to me. I will escort Hai ships here, pretending that if I do you will stop attacking them."`
```

Comment:
Is this a situation where you're luring Hai ships to pirates, so that they can attack and pillage them? 
```suggestion
				`	"That seems like a good deal to me. I will escort Hai ships here, pretending that if I'm with them you will stop attacking them."`
```